Library/Homebrew/service.rb

```diff
@@ -56,6 +58,10 @@ def initialize(formula, &block)
       @sockets = T.let({}, Sockets)
       @working_dir = T.let(nil, T.nilable(String))
       instance_eval(&block) if block
+
+      if @nice&.negative? && !requires_root?
+        raise TypeError, "Service#nice: negative nice values require root access. Set require_root: true"
+      end
```

Comment:
I didn’t make it a one-liner because of:

```
service.rb:62:105: C: [Correctable] Style/IfUnlessModifier: Modifier form of if makes the line too long.
      raise TypeError, "Service#nice: negative nice values require root access. Set require_root: true" if @nice&.negative? && !requires_root?
```

Reply:
Well whoops:

```
service.rb:62:7: C: [Correctable] Style/GuardClause: Use a guard clause (return unless @nice&.negative? && !requires_root?) instead of wrapping the code inside a conditional expression.
      if @nice&.negative? && !requires_root?
      ^^
```

I’ll move it to a helper function in that case.